Formula/millet.rb

```diff
@@ -0,0 +1,92 @@
+require "json"
+
+class Millet < Formula
+  desc "Language server for Standard ML (SML)"
+  homepage "https://github.com/azdavis/millet"
+  url "https://github.com/azdavis/millet.git",
+      tag:      "v0.9.4",
+      revision: "f99f096471887deae67e3569b30df07aa7b33acb"
```

Comment:
why not use `https://github.com/azdavis/millet/archive/refs/tags/v0.9.4.tar.gz`?

Reply:
@chenrui333 That's what I had at first, but seeing how everything is based on git around here, I thought maybe this was preferable. Is it preferable then to link to the tarball?

Reply:
yeah, that is right, we'd like to use tarball whenever possible